lighthouse-core/gather/gather-runner.js

```diff
@@ -29,8 +29,22 @@ const UIStrings = {
    */
   warningTimeout: 'The page loaded too slowly to finish within the time limit. ' +
   'Results may be incomplete.',
+  /**
+   * @description Warning that the host device where Lighthouse is running appears to have a slower
+   * CPU than the expected Lighthouse baseline.
+   */
+  warningSlowHostCpu: 'The device that ran this test appears to have a slower CPU than  ' +
+  'Lighthouse expects. This can negatively affect your performance score. Learn more about using ' +
+  '[custom throttling settings](https://github.com/GoogleChrome/lighthouse/blob/ccbc8002fd058770d14e372a8301cc4f7d256414/docs/throttling.md#calibrating-multipliers) ' +
```

Comment:
```suggestion
  '[custom throttling settings](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#calibrating-multipliers) ' +
```

while it adds a risk of this 404'ing, i think it's better that we can update the text people are reading.

Reply:
:/ makes me nervous for all the old reports left out there, but I believe in you :)